dev_guide/managing_images.adoc

```diff
@@ -359,7 +359,7 @@ metadata:
 ====
 ifdef::openshift-enterprise,openshift-origin[]
 [[using-is-with-k8s]]
-== Using Image Streams with Kubernetes Resources (Technology Preview)
```

Comment:
@mfojtik @soltysh Fair to drop the "(Technology Preview)" here? Looks like it was added in 3.6 via https://github.com/openshift/openshift-docs/pull/4905.

Reply:
Yup.